debian/make-preseed-iso.sh

```diff
@@ -6,7 +6,7 @@ set -e
 
 function extract_iso() {
   echo "Extracting iso: $1..."
-  rm -rf isofiles
+  [ -d isofiles ] && echo "Removing temp files. This must be done as root"; sudo rm -rf isofiles
```

Comment:
`;` needs to be `&&` for for the `rm` to conform to the test result otherwise its equivalent to having it on the next line